tests/test_entry.py

```diff
@@ -1,12 +1,78 @@
+import os
 from pathlib import Path
-from renamer.entry import rename_variable
+from renamer.entry import rename_class_or_function, move_class_or_function
 
 
-def test():
-    got = rename_variable(
-        Path('tests/fixtures/source.py').read_text(),
-        'arg1',
-        'new_name',
+def replace_last_folder(path, new_folder_name):
+    path = Path(path)
+    path_components = list(path.parts)
+    for i in range(len(path_components) - 1, -1, -1):
+        if '.' not in path_components[i]:
+            path_components[i] = new_folder_name
+            break
+    new_path = Path(*path_components)
+    return new_path
```

Comment:
```suggestion
def replace_last_folder(path, new_folder_name):
    path = Path(path)
    return path.parent.parent / new_folder_name / path.name
```
Очень тяжело читается с циклом